app/controllers/foreman_wreckingball/hosts_controller.rb

```diff
@@ -108,8 +108,27 @@ def triggering_params
       end
     end
 
-    def find_status
-      @status = HostStatus::Status.find_by!(:id => params[:status_id], :host_id => @host.id)
+    def statuses_params
+      @statuses_params ||= params.permit(:host_association, :owned_only, status_ids: [])
+    end
+
+    def find_statuses
+      @statuses = begin
+        host_association = statuses_params[:host_association]&.to_sym
```

Comment:
"Avoid usage of to_sym on user-supplied input."
https://www.sitepoint.com/rails-security-pitfalls/

Does this apply here?